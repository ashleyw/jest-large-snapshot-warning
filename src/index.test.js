const { setMaxSnapshotSize } = require("./");

setMaxSnapshotSize(3);

test("large snapshot", () => {
  expect({ a: 1, b: 2 }).toMatchSnapshot();
});

test("large snapshot (inline)", () => {
  expect({ a: 1, b: 2 }).toMatchInlineSnapshot(`
    WARNING: THIS IS A LARGE SNAPSHOT!

    Object {
      "a": 1,
      "b": 2,
    }
  `);
});

test("small snapshot", () => {
  expect({ a: 1 }).toMatchSnapshot();
});

test("small snapshot (inline)", () => {
  expect({ a: 1 }).toMatchInlineSnapshot(`
    Object {
      "a": 1,
    }
  `);
});
