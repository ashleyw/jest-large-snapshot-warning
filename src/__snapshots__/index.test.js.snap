// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`large snapshot 1`] = `
WARNING: THIS IS A LARGE SNAPSHOT!

Object {
  "a": 1,
  "b": 2,
}
`;

exports[`small snapshot 1`] = `
Object {
  "a": 1,
}
`;
